app/mailers/user_mailer.rb

```diff
@@ -217,10 +254,14 @@ def related_work_notification(user_id, related_work_id)
     @related_work = RelatedWork.find(related_work_id)
     @related_parent_link = url_for(:controller => :works, :action => :show, :id => @related_work.parent)
     @related_child_link = url_for(:controller => :works, :action => :show, :id => @related_work.work)
-    mail(
-      :to => @user.email,
-      :subject => "[#{ArchiveConfig.APP_SHORT_NAME}] Related work notification"
-    )
+    I18n.with_locale(Locale.find(@user.preference.prefered_locale).iso) do
+      mail(
+        to: @user.email,
+        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Related work notification"
+      )
+   end
```

Comment:
`end` at 262, 3 is not aligned with `I18n.with_locale(Locale.find(@user.preference.prefered_locale).iso) do` at 257, 4